decidim-core/app/assets/images/decidim/icons.svg

```diff
@@ -236,4 +236,5 @@
   <symbol viewBox="0 0 8 8" id="icon-zoom-in"><title>zoom-in</title> <path d="M3.5 0c-1.93 0-3.5 1.57-3.5 3.5s1.57 3.5 3.5 3.5c.61 0 1.19-.16 1.69-.44a1 1 0 0 0 .09.13l1 1.03a1.02 1.02 0 1 0 1.44-1.44l-1.03-1a1 1 0 0 0-.13-.09c.27-.5.44-1.08.44-1.69 0-1.93-1.57-3.5-3.5-3.5zm0 1c1.39 0 2.5 1.11 2.5 2.5 0 .59-.2 1.14-.53 1.56-.01.01-.02.02-.03.03a1 1 0 0 0-.06.03 1 1 0 0 0-.25.28c-.44.37-1.01.59-1.63.59-1.39 0-2.5-1.11-2.5-2.5s1.11-2.5 2.5-2.5zm-.5 1v1h-1v1h1v1h1v-1h1v-1h-1v-1h-1z"/> </symbol>
   <symbol viewBox="0 0 8 8" id="icon-zoom-out"><title>zoom-out</title> <path d="M3.5 0c-1.93 0-3.5 1.57-3.5 3.5s1.57 3.5 3.5 3.5c.61 0 1.19-.16 1.69-.44a1 1 0 0 0 .09.13l1 1.03a1.02 1.02 0 1 0 1.44-1.44l-1.03-1a1 1 0 0 0-.13-.09c.27-.5.44-1.08.44-1.69 0-1.93-1.57-3.5-3.5-3.5zm0 1c1.39 0 2.5 1.11 2.5 2.5 0 .59-.2 1.14-.53 1.56-.01.01-.02.02-.03.03a1 1 0 0 0-.06.03 1 1 0 0 0-.25.28c-.44.37-1.01.59-1.63.59-1.39 0-2.5-1.11-2.5-2.5s1.11-2.5 2.5-2.5zm-1.5 2v1h3v-1h-3z"/> </symbol>
+  <symbol id="icon-verification-badge" viewBox="0 0 12 12"><title>verification-badge</title> <g fill="none" fill-rule="evenodd"> <circle fill="#EC6A5B" cx="6" cy="6" r="6"/> <path fill="#FFF" d="M5.943 7.689l-1.926.906.267-2.112L2.826 4.93l2.092-.4 1.025-1.865L6.97 4.531l2.091.4-1.457 1.552.267 2.112z"/> </g> </symbol>
```

Comment:
The verification-badge icon is already integrated in the icons file, I should've mentioned that. The id is `icon-verified-badge`.

Reply:
Thx! Then I'll remove it, no worries